libs/cdk/render-strategies/src/lib/README.md

```diff
@@ -0,0 +1,181 @@
+# RxStrategyProvider
+
+`RxStrategyProvider` is the best way to consume full power of concurrent strategies to schedule work that is not related to rendering.
+
+> Want to play with it? Here's [demo link](https://stackblitz.com/edit/angular-ivy-1vfpoe) 
+
+## Why to schedule
+
+Chromium based browsers considers all tasks that taking more than 50ms as long tasks. If task runs more than 50ms, users will start noticing lags. Optimally all user interactions should happen at 30 fps framerate with 32ms budget per browser task. In ideal world it should be 60 fps and 16ms budget. 
+
+> 💡 In reality browser has a reserved overhead of 4ms, try to stick to 28ms of work for 30 fps and 12ms for 60 fps.
+
+## Scheduling mechanisms in browser
+
+There are multiple ways to schedule task in the browser. 
+
+- `setTimeout`
+- `requestAnimationFrame`
+- `requestIdleCallback`
+- `Promise.resolve`
+- `queueMicrotask`
+
+None of them has a full notion about what happens in the browser and can be unreliable for performance optimizations.
+
+## Concurrent strategies
+
+> 💡 Under the hood all our concurrent strategies are based on MessageChannel technology.
+
+To address the problem of long tasks and help browser split the work @rx-angular/cdk provides  concurrent strategies. This strategies will help browser to chunk the work into non-blocking tasks whenever it's possible. 
+
+### Core concepts
+
+**Frame budget**
+
+Each strategy knows the budget and knows which task will go above this budget.
+
+**Chunking**
+
+Strategies will split work into non-blocking pieces whenever it's possible.
+
+**Deadline**
+
+Some of the strategies has limited amount of time to do tasks in chunks, after this time all remaining work will go as a single task.
+
+**Priority**
+
+Each strategy has its own queue with its own priority.
+
+### Available concurrent strategies
```

Comment:
Maybe a cross-link to render-strategy docs here?

Reply:
Done